webapp/components/wpt-results.html

```diff
@@ -281,9 +275,8 @@ <h3>
           },
           displayedTests: {
             type: Array,
-            value: []
-          },
-          // Users request to show a diff column.
+            computed: 'computeDisplayedTests(searchResults)',
+          },// Users request to show a diff column.
```

Comment:
The comment should go to a new line.

Reply:
Just fixed this locally. Letting https://travis-ci.org/web-platform-tests/wpt.fyi/builds/419660483 follow through before pushing.